module-4/backend/week_four.md

```diff
@@ -4,17 +4,65 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-* What's the purpose of a project management tool?
+
+* What's the purpose of a project management tool?  
+    To organize the project into small managable parts
+    To keep track of what has already been done and what needs to be done 
+    To keep track of changes of the progress of each part including potentially the conversation around the progress
+* What are the stages of a card when using Pivotal Tracker? 
+    icebox
+    current iteration
+    up next
+    done
+* What are the four types of cards available in Pivotal Tracker? 
+    started 
+    finished 
+    delivered 
+    accepted
 * What's your favorite project management tool?
-* Why do we create staging environments?
+    Waffle :3
+* Why do we create staging environments? 
+    so that we can work with the production environment (different api keys + variable names + databases) in a way that does      
+    not effect production in anyway. 
 * What are the characteristics of a good README (in your opinion)?
-* What's one main improvement you're going to make to your code regarding accessibility issues?
-* What are some basic security concerns to be aware of when building applications?
-* Why is continuous integration helpful/important?
+    - A good explaination of how you use the tool 
+    - Listing any endpoints and how you interact with them 
+    - Making it begginner friendly 
+    - Explains how the application was build, what tools were used
+* What's one main improvement you're going to make to your code regarding accessibility issues? 
+    - Make sure you can read through the whole of the available pages and the text to voice program pick up 
+      all the intended text 
+    - Making the webpages calm and minimal as possible
+* What are some basic security concerns to be aware of when building applications? 
+    - How you're setting up the login/logout process -> 
+            
+            - don't store passwords in plain text
+            - at registration that users make passwords strong
+            - using strong params when using incoming data
+            - sanitizing the incoming data 
+            - keeping certain information secure -> whether some pages do or do not exist, if the login was incorrect 
+              revealing which ones were wrong
+          
+* Why is continuous integration helpful/important? 
+    contineous integration is helpful so that you can test the application in a different environment
 * What are some continuous integration tools?
+    Travis CI
 
 #### Review  
 
 * What are some characteristics of "good" git workflow?
+    small commit messages
```

Comment:
I'm not sure this is always the case. The first line should always be a quick summary, but it is totally fine (and sometimes necessary) to add details in the body.